### ISSUE TITLE
PWA: shortcuts like ctrl-W are sometimes handled by the browser

### ISSUE THREAD

**Issue by laurentlb** [microsoft/vscode#150735]
Does this issue occur when all extensions are disabled?: Yes

- VS Code Version: 1.67.2
- Chrome Version: 101
- OS Version: Chrome OS (probably others too)

Steps to Reproduce:

1. Open https://vscode.dev
2. Install as a PWA
3. When an editor has the focus, shortcuts like `Ctrl-W` or `Ctrl-N` are correctly handled by vscode.
4. Open an image or a webview, try again `Ctrl-W` or `Ctrl-N`. It's handled by the browser.

At Google, we've had multiple reports like this. `Ctrl-W` will sometimes close a tab (as expected); it will sometimes close the full app (which can be very annoying). I've reproduced the same issue with `Ctrl-N`: depending on what has the focus, Chrome might open a new browser tab or not.

**Comment by a-stewart** [comment]
To add a bit more context, this occurs when the user is focused in a webview. Whilst VS Code core handles cmd+w, it seems like whilst in a webview, keyboard shortcuts are not intercepted before the browser gets them.

**Comment by xaethos** [comment]
As a workaround, and maybe helpful debug data:

I was running into this issue every single time until I changed the setting [`window.confirmBeforeClose`](https://code.visualstudio.com/updates/v1_51#_prevent-accidental-close) from `never` to `keyboardOnly`. After that, I got a single "close" confirmation the first time I tried closing an editor tab with `ctrl-w`, and from then on editor tabs are closing correctly without any confirmation. I can also use `ctrl-w` in the terminal without confirmation; before it would close the entire app.

The "fix" also persists if I restart the entire browser (Chrome, not just the PWA).
